arduino-ide-extension/src/browser/serial/monitor/monitor-widget.tsx

```diff
@@ -74,6 +74,10 @@ export class MonitorWidget extends ReactWidget {
     this.monitorManagerProxy.startMonitor();
   }
 
+  protected override onAfterAttach(msg: Message): void {
+    this.monitorManagerProxy.setMonitorWidgetStatus(this.isAttached);
```

Comment:
Hi, when overriding methods from `Widget`, please make sure to call the `super.theOverriddenMethod(msg);` . Usually, it does not matter whether you call `super` first or last, but `super` must be called. Otherwise, the framework and the underlying widget tracker might miss events.  Please reference, for example, the [`ListWidget`](https://github.com/arduino/arduino-ide/blob/fe3fbb189c90ce7e0141977061dabe817ac824a9/arduino-ide-extension/src/browser/widgets/component-list/list-widget.tsx#L25) how its overridden methods call `super`.

In this monitor widget class, I noticed the same defect in `onBeforeAttach` a couple of weeks ago.

Reply:
You are right. I'm still working on it and maybe overriding `onAfterAttach` is no longer needed. In any case I can add the call of super in `onBeforeAttach`. Thanks @kittaakos